### PR TITLE
[ko]Adopt new localization string for release page

### LINKED ISSUE #36759: Adopt new localization string for releases page
**What would you like to be added**
This issue is a follow-up task from https://github.com/kubernetes/website/issues/36568.

https://kubernetes.io/ko/releases/ displays some English text: 

image

To replace this text with localized strings, you can update `data/i18n/ko/ko.toml` with [these recent changes](https://github.com/kubernetes/website/commit/b6299f907b19eaacb71391d264af5c229cb082aa) from `data/i18n/en/en.toml`.  Specifically, these:

```
# Avoid using conjunction_1.
# Must match the context in layouts/shortcodes/release-data.html
# Appears on https://kubernetes.io/releases/
# For example the "and" in "Complete 1.25 Schedule and Changelog"
[conjunction_1]
other = "and"
```
```
# Deprecated. Planned for removal in a future release.
# Use [release_full_details_initial_text] instead.
[release_complete]
other = "Complete"

# Replace [release_complete] with [release_full_details_initial_text]
[release_full_details_initial_text]
other = "Complete"

[release_schedule]
other = "Schedule"

[release_changelog]
other = "Changelog"
```

### PR DESCRIPTION
**Pull request by jinnypark9393**
closes #36759
- I also updated Line 38~65 since it has been reflected in data/i18n/ko/ko.toml yet.

### CODE REVIEW

File: data/i18n/ko/ko.toml [hunk 1]
```diff
@@ -234,6 +270,21 @@ other = "("
...
 other = "2006-01-02"
 
+# Deprecated. Planned for removal in a future release.
+# Use [release_full_details_initial_text] instead.
+[release_complete]
+other = "완료"
+
+# Replace [release_complete] with [release_full_details_initial_text]
+[release_full_details_initial_text]
+other = "완료"
```
Comment (seokho-son) [inline data/i18n/ko/ko.toml:280]:
```suggestion
other = "전체"
```
image

I think this string means "complete details" :)

**Review by seokho-son — COMMENTED** [review]
Hi @jinnypark9393 

This contribution is very important for Korean localization !
Thanks !

Please update this PR according to the comment below.

File: data/i18n/ko/ko.toml (hunk 1 above)
Comment (jinnypark9393) [reply data/i18n/ko/ko.toml:280]: It makes sense! I was wondering why should we put "complete(완료)" right before the release version 😅 Let me fix this right away.

**Comment by seokho-son** [comment]
Hi @jinnypark9393 
Thank you for the update!!

Netlify build has been failed due to an occasional error from Netlify itself.
https://github.com/kubernetes/website/issues/35780

So, could you help change the current commit (change the commit hash) by using `git commit --amend`. (and push)

**Comment by seokho-son** [comment]
Thanks @jinnypark9393 @rolfedh !

/lgtm

**Pull request merged**